### DISCUSSION TITLE
如何通过代码调用下一集这个按钮，因为控制条要隐藏，自己封装了一个进度条

### DISCUSSION THREAD

**Discussion by XiaotaoTTTtian** [bytedance/xgplayer#819] [Q&A]
**您使用的西瓜播放器版本是多少？2.32.0

**您使用的操作系统和浏览器分别是？ window11   chrome

**如何复现问题？ What did you do?**

**您期望的播放器正常行为是？ What did you expect to happen?**

**实际播放器的表现是？ What actually happened?**

**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 实现一个一天的视频回放

**Comment by gemxx** [comment]
可以参照[playNext源码](https://github.com/bytedance/xgplayer/blob/master/packages/xgplayer/src/controls/playNext.js)自定义实现，但不建议自定义这里
